src/fmu/dataio/_utils.py

```diff
@@ -361,6 +361,11 @@ def some_config_from_env(envvar: str = "FMU_GLOBAL_CONFIG") -> dict | None:
     return ut.yaml_load(cfg_path, loader="fmu")
 
 
+def read_named_envvar(envvar: str) -> str | None:
+    """Read a specific (named) environment variable."""
+    return os.environ.get(envvar, None)
+
+
```

Comment:
Do we need a separate routine for this? (unless we want to make more checks etc, then the oneliner `os.environ.get()` will do)

Reply:
Possibly not, but it made some sense to me to have a dedicated unit test on it. But I made that decision before I knew it would become just 1 line...